README.md

```diff
@@ -99,22 +99,28 @@ concurrently (5200 blob downloads), with 300MB/s speed limit on all agents (usin
 
 # Usage
 
+All Kraken components can be deployed as Docker containers. To build the Docker images:
+```
+$ make images
+```
+For information about how to configure and use Kraken, please refer to the [documentation](docs/CONFIGURATION.md).
+
 ## Kraken on Kubernetes
 
-You can use our example Helm chart to deploy Kraken (with an example http fileserver backend) on your k8s cluster:
+You can use our example Helm chart to deploy Kraken (with an example http fileserver backend) on
+your k8s cluster:
 ```
 helm install --name=kraken-demo ./helm
```

Comment:
Add `$` in there if you don't mind, just so its consistent with the rest of the readme.

Reply:
ok